e2e/src/usecases/DappListRecent.js

```diff
@@ -0,0 +1,158 @@
+import { reloadReactNative } from '../utils/retries'
+import { sleep, waitForElementId, getElementText, getElementTextList } from '../utils/utils'
+
+const jestExpect = require('expect')
+
+/**
+ * From the home screen, navigate to the dapp explorer screen
+ * Disable detox device synchronization to allow for dapp list to load
+ * https://github.com/wix/Detox/issues/2799
+ */
+async function navigateToDappList() {
+  await device.disableSynchronization()
+  await waitForElementId('Hamburger')
+  await element(by.id('Hamburger')).tap()
+  await waitForElementId('dapps-explorer-icon')
+  await element(by.id('dapps-explorer-icon')).tap()
+  await waitFor(element(by.id('DAppExplorerScreen/loading')))
+    .not.toBeVisible()
+    .withTimeout(30 * 1000)
```

Comment:
i think this is racy because the loader is not guaranteed to be there when the screen first loads, i see in the last test there is an assertion like `await waitFor(element(by.id('DAppExplorerScreen/DappsList')))
      .toExist()
      .withTimeout(10 * 1000)` to see that the dapps list exists, would that be a better assertion here too?

Reply:
Commented in https://github.com/valora-inc/wallet/pull/3287#discussion_r1062786920!